### PR TITLE
Added responsive nav CSS & JS, updated styling

### PR DESCRIPTION
**Pull request by Brazill-Mitchell**
I've updated the nav menu to collapse when resizing, as well as some javascript for the navigation button to toggle the menu. Also, I added "btn-article: hover" CSS for your "Latest from Atomist" section @corochena.
Aside from that, I added a little bit of styling to head.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,38 +12,77 @@
...
+      <script>
+          let navVisible = false;
+          function toggleNav(){
+              let nav = document.getElementById("nav-side");
+              let navButton = document.getElementById("nav-button");
+              if (navVisible == false){
+                  console.log("nav on");
+                  nav.style.visibility = "visible";
+                  navButton.innerHTML = "×";
+                  navVisible = true;
```
Comment (sarajoha) [inline index.html:45]: Maybe for the next sprint it'll be better to have Js in a separate file, to keep the code clean.

File: index.html [hunk 2]
```diff
@@ -12,38 +12,77 @@
...
+<!-- Collapsed Navigation -->
+      
+<div class="position-fixed" style="width: 100%;height:100vh;z-index:20;visibility: collapse" id="nav-side">
+    <img src="src/assets/atomist_logo_shapes_banner-home.jpg" style="position:absolute;z-index:-10;left:-1500px;">
+    <img src="src/assets/Atomist-Logo-White-Horiz.png" style="width: 110px" class="m-3">
+    <div class="container h-100 ml-0 d-flex flex-column" style="z-index:10">
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Developer</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Pricing</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Docs</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Blog</div>
```
Comment (sarajoha) [inline index.html:26]: I see that the style `z-index:20` is used several times, I'd suggest you put it as a class or variable in the css file.

File: index.html [hunk 3]
```diff
@@ -12,38 +12,77 @@
     <title>Atomist | Software delivery information for cloud-native applications</title>
   </head>
   <body>
-
+      
+<!-- Collapsed Navigation -->
+      
+<div class="position-fixed" style="width: 100%;height:100vh;z-index:20;visibility: collapse" id="nav-side">
+    <img src="src/assets/atomist_logo_shapes_banner-home.jpg" style="position:absolute;z-index:-10;left:-1500px;">
```
Comment (sarajoha) [inline index.html:20]: As a general rule, I think it's best to limit the number of `style` we use in the html. For example, you could take everything that's on the `style`, inside: `<div class="position-fixed" style="width: 100%;height:100vh;z-index:20;visibility: collapse" id="nav-side">` and put it in a css class, and then you would only need to refer to the class name in the html.

File: index.html [hunk 4]
```diff
@@ -12,38 +12,77 @@
...
+    <img src="src/assets/Atomist-Logo-White-Horiz.png" style="width: 110px" class="m-3">
+    <div class="container h-100 ml-0 d-flex flex-column" style="z-index:10">
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Developer</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Pricing</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Docs</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Blog</div>
+            <div id="login-collapsed" class="nav-collapsed  row-1">Log in</div>
+            <button class=" text-nowrap btn btn-primary btn-lg" onclick="window.location.href = 'https://app.atomist.com/user/signup?_ga=2.246771340.948132213.1553277995-6570700.1552576324';" id="btn-start-collapsed" >Get Started</button>
+    </div>
```
Comment (sarajoha) [inline index.html:29]: I was thinking we can put the interactivity in a js file.

**Review by sarajoha — APPROVED** [review]
I left some commentaries, let me know what you think. :)

**Review by corochena — APPROVED** [review]
Hey @brazill91 , for the next sprint we can concentrate in separate the style in the css file and the behavior in the js file so we have the html clean. Good job!

**Pull request merged**